utils/models/error.py

```diff
@@ -0,0 +1,113 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, field_validator
+
+from integration.tests.basic.helpers.errors import Error32602
+from utils.models.model_types import (
+    ErrorCodeField,
+    IdField,
+    JsonRPCString,
+    NotSupportedMethodString,
+    RequiredParamsString,
+)
+
+
+class EthErrorData(BaseModel):
+    errors: List[str]
+
+    @field_validator("errors")
+    def check_errors_list(cls, value):
+        if len(value) == 0:
+            raise ValueError("errors must be in a list")
+
+    class Config:
+        extra = "forbid"
```

Comment:
Could it be a good idea to use inheritance for model configuration?

```suggestion

class ForbidExtra(BaseModel):
    model_config = ConfigDict(extra='forbid')

class EthErrorData(ForbidExtra):
    errors: List[str]

    @field_validator("errors")
    def check_errors_list(cls, value):
        if len(value) == 0:
            raise ValueError("errors must be in a list")
```

Reply:
fixed ✅